pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -281,3 +307,68 @@ func convertToEnvoyJwtConfig(jwtRules []*v1beta1.JWTRule) *envoy_jwt.JwtAuthenti
 		Providers: providers,
 	}
 }
+
+// GetMutualTLSMode returns the MutualTLSMode enum corresponding to the given peer authentication policy.
+// If the input authentication is nil, returns MTLSPermissive.
+func GetMutualTLSMode(peer *v1beta1.PeerAuthentication) model.MutualTLSMode {
+	if peer == nil || peer.Mtls == nil {
+		return model.MTLSPermissive
+	}
+	switch peer.Mtls.Mode {
+	case v1beta1.PeerAuthentication_MutualTLS_DISABLE:
```

Comment:
and what about the unset thing?

Reply:
Good point. Now it will be more complicated :)

Reply:
It is actually simple if you use the same logic as the Sidecar